src/main/recorder.ts

```diff
@@ -280,6 +289,52 @@ type RecorderOptionsType = {
         obsRecorder.reconfigure(this._options);
         if (mainWindow) mainWindow.webContents.send('refreshState');
     }
+
+    /**
+     * Generate a filename for the final video of a recording according
+     * to its category, if possible.
+     *
+     * The final filename should contain adequate information to quickly be able
+     * to identify the video files on the filesystem and be able to tell what it
+     * was about and the result of it.
+     */
+    getFinalVideoFilename (metadata: Metadata): string | undefined {
+        let outputFilename = '';
+        let zoneName: string;
+        const resultText = getVideoResultText(metadata.category, metadata.result);
+
+        switch (metadata.category) {
+            case VideoCategory.TwoVTwo:
```

Comment:
Can we use default to avoid this pattern of multiple cases in a row

Reply:
We can - I know I did that somewhere else, but for this one I just felt it made more sense to have the explicitly stated to make it quickly obvious what was being handled how, but ultimately I don't really care either way =)